scripts/wireguard/pivpn.sh

```diff
@@ -80,6 +92,8 @@ showHelp(){
     echo ":::  -l,  list             List all clients"
     echo ":::  -qr, qrcode           Show the qrcode of a client for use with the mobile app"
     echo ":::  -r,  remove           Remove a client"
+    echo ":::  -off,  off            Disable a user"
+    echo ":::  -on ,  on             Enable a user"
```

Comment:
Add space to the lines above and below these so the commas and the text line up vertically